app/controllers/ideas_controller.rb

```diff
@@ -33,6 +33,9 @@ def show
   private
 
   def idea_params
-    params.require(:idea).permit(:name, :description, :picture)
+    params
+      .require(:idea)
+      .permit(:name, :description, :picture)
+      .merge(user: current_user)
```

Comment:
I's be a bit more explicit with current user and use 

current_user.ideas.new (idea params) in `new` action 🤔 .

However, there is nothing wrong with this approach, so take it or leave it :)

Reply:
Really good point! @korneliacieslik would you mind to take a look ? :)

Reply:
@nidalaa  it is fixed now. Could you take a look?